rpm/ocrdesktop/09-keybindings-ocrdesktop

```diff
@@ -0,0 +1,5 @@
+[org/mate/desktop/keybindings/custom9]
+action='/opt/ocrdesktop/ocrdesktop -l eng'
```

Comment:
```suggestion
action='/usr/bin/ocrdesktop -l eng'
```
Hello @kk-at-redhat  and thank you for the update. You have just left this old path here, other than that it looks great! Thank you.

Reply:
Thank you, fixed.